subprojects/docs/src/docs/userguide/dep-man/03-controlling-transitive-dependencies/platforms.adoc

```diff
@@ -174,6 +174,8 @@ In addition to libraries, version catalog supports declaring plugin versions.
 While libraries are represented by their group, artifact and version coordinates, Gradle plugins are identified by their id and version only.
 Therefore, they need to be declared separately:
 
+WARNING: Version catalogs do not support Settings plugins.
```

Comment:
I think the warning is a bit confusing. You can (and should) define version catalogs in a settings plugin. What is not supported is using a plugin defined in a version catalog in the settings, so I would suggest to rephrase:

> WARNING: You cannot use a plugin declared in a version catalog in your settings file or settings plugin (because catalogs are defined in settings themselves, it would be a chicken and egg problem).

Reply:
Much better 👍 Thanks!